### DISCUSSION TITLE
Accidentals and <nc> elements

### DISCUSSION THREAD

**Discussion by pfefferniels** [music-encoding/music-encoding#1402] [Q&A] [answered]
Is there a reason that the`<nc>` element only allows `accid.ges` attributes, but not `accid`? The background of my question is that in some instances accidentals do indeed appear visually:
![grafik](https://github.com/music-encoding/music-encoding/assets/47777840/62a356b8-7810-4227-83e3-7457bb862278)
(Example for a ♭ in B-Afosb ms. 9)

**Comment by ahankinson** [comment]
Can you use the `<accid>` element?

**Reply by pfefferniels** [reply]
Hm, if I don't miss anything `<accid>` is not allowed as a child of `<nc>` (see [here](https://music-encoding.org/guidelines/v5/elements/nc.html#mayContain_compact_tab)) …

**Reply by rettinghaus** [reply, marked as answer]
In neumes customization accidentals are expected to be encoded as a layer element (between neumes).

**Reply by pfefferniels** [reply]
Oh, I see! Thanks a lot :-)
